aspnetcore/migration/20_21.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Migrate from ASP.NET MVC to ASP.NET Core MVC
+author: rick-anderson
+description: This article covers the basics of migrating an ASP.NET Core 2.0 app to 2.1.
+manager: wpickett
+ms.author: riande
+ms.date: 5/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: migration/20_21
+---
+# Migrate from ASP.NET Core 2.0 to 2.1
+
+By [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+This article:
+
+* Covers the basics of migrating an ASP.NET Core 2.0 app to 2.1.
+* Provides an overview of the changes to the ASP.NET Core web application templates.
+
+A quick way to get an overview of the changes in 2.1 is to:
+
+* Create an ASP.NET Core 2.0 web app named WebApp1.
+* Commit the WebApp1 in a source control system.
+* Delete WebApp1 and create an ASP.NET Core 2.1 web app named WebApp1 in the same place.
+* Review the changes in the 2.1 version.
+
+This article provides an overview on migration to  ASP.NET Core 2.1. It does not contain a complete list of all changes needed to migrate to version 2.1. Some projects might require more steps depending on the options selected when the project was created and modifications made to the project.
+
+## Update the project file to use 2.1 versions
+
+Update the *.csproj* project file:
```

Comment:
Why isn't there a before and after picture for the project file migration as well?

Reply:
@davidfowl I added the before/after *csproj* files. Also added the before/after pic which I think is a distraction and we should remove. It's so lit up it doesn't tell you much.